src/components/Button/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Button.css';
+
+const Button = (props) => {
+	const styles = {
```

Comment:
We are already importing `Button.css`, can we just transfer these styles onto that file?

Reply:
Usually I would, but I need js here. I can use classes instead of defining styles like this, don't think it would look much cleaner though.

Reply:
Yeah the styles are conditional based on what kind of button we have. So instead of placing all these styles here we can conditionally give the button element  a `className` attribute. For instance, if it's supposed to be an orange button then the classname of the button should be `orange` or something. So we can consistently keep all styling in the CSS files. 

```javascript

const colorClass = props.color ? 'button orange' : 'button';
const wideClass = props.width ? `${colorClass} wide` : `${colorClass}`

<button onClick={() => props.handleClick(name)} style={styles} className={`${wideClass}`}>{name}</button>```

Then in your CSS files you will have a default background-color for your buttons and then for the 'orange' class you will of course set a background-color of orange.

In your CSS files also you can have a default width of '25%' for all buttons but for the 'wide

The benefit of this is it makes your code more consistent. Yes while you can technically include all styles inline in React, it makes it easier to reason about when we separate our styles from our component files.